README.md

```diff
@@ -25,122 +25,95 @@ Apollo-android features:
 
 The latest Gradle plugin version is [ ![Download](https://api.bintray.com/packages/apollographql/android/apollo/images/download.svg) ](https://bintray.com/apollographql/android/apollo-gradle-plugin/_latestVersion)
 
-To use this plugin, add the dependency to your project's root build.gradle file:
+In your app Gradle file, apply the `com.apollographql.apollo` plugin and add the Apollo dependencies:
```

Comment:
I don't think it needs to be app. What about "In your Gradle module's build.gradle(.kts) file"

Reply:
Yep. I chose app because I think there were other references but more generic works for me too.